### PR TITLE
Add doc for woketo client

### PR DESCRIPTION
**Pull request by Nek-**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -64,10 +73,10 @@ $server->setMessageHandler(new YourMessageHandler(), '/path');
 $server->start();
 ```
 
-The `Websocket` instantiation take the following parameters:
+The `WebsocketServer` instantiation take the following parameters:
```
Comment (folliked) [inline docs/index.md]: takes ?

File: docs/index.md [hunk 2]
```diff
@@ -78,6 +87,52 @@ The `Connection` object has the following methods you can use:
...
+
+You can then run your client:
+
+```php
+$client = new WebSocketClient('ws://127.0.0.1:9000/foobar', []);
+$client->start();
+```
+
+
+The `WebsocketClient` instantiation take the following parameters:
```
Comment (folliked) [inline docs/index.md]: takes

File: docs/index.md [hunk 3]
```diff
@@ -95,6 +150,8 @@ implement it.
 Configuration Reference
 -----------------------
 
+### Server configuration
+
 The configuration is split in some parts:
 - The `frame` key configure the way Frames are managed
 - The `message` key configure the way Frames are stacked
```
Comment (folliked) [inline docs/index.md]: configures

File: docs/index.md [hunk 4]
```diff
@@ -95,6 +150,8 @@ implement it.
 Configuration Reference
 -----------------------
 
+### Server configuration
+
 The configuration is split in some parts:
 - The `frame` key configure the way Frames are managed
```
Comment (folliked) [inline docs/index.md]: configures

File: CHANGELOG.md [hunk 5]
```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [BC Break] Message handlers now use an `AbstractConnection` class in their methods
 - **Internal:** the method `processHandcheck` is now `processHandshake` and follows `AbstractConnection` class requirements
 - **Internal:** the `MessageProcessor` now needs to be aware of its quality of client or server.
+- **Internal:** the `BitManipulation` class now uses PHP native functions
+- **Internal:** the `BitManipulation` class use new method names
```
Comment (folliked) [inline CHANGELOG.md]: I just saw this one oops : useS

File: docs/index.md [hunk 6]
```diff
@@ -78,6 +87,52 @@ The `Connection` object has the following methods you can use:
...
+class MyMessageHandler implements MessageHandlerInterface
+{
+    public function onConnection(AbstractConnection $connection)
+    {
+        // This method is called when a new client is connected to your server
+    }
+    
+    public function onMessage(string $data, AbstractConnection $connection)
+    {
+        // This method is called when a text message is send
```
Comment (valanz) [inline docs/index.md]: [...] is sent

**Pull request merged**